### PR TITLE
Removed screenshots that were outdated or didn't have value + reworke…

### PR DESCRIPTION
**Pull request by kodster28**
Addresses the following JIRAs:
- PCX-373
- PCX-977

Also reworked main tutorial for creating via UI + added more info about configurations to load balancers.

### CODE REVIEW

**Comment by kodster28** [comment]
For visuals of the two major page changes.

**Understand basics > Load balancers**
image

**Create via dashboard UI**
image

File: products/load-balancing/src/content/understand-basics/load-balancers.md [hunk 1]
```diff
@@ -10,6 +10,36 @@ A Cloudflare load balancer is identified by the DNS hostname whose traffic you w
...
+
+An **active-active failover** distributes traffic to servers in the same pool until the pool reaches its failure threshold (configurable). At the point of failure, your load balancer would then re-direct traffic to the **fallback pool**.
+
+This setup speeds up overall requests, but is more vulnerable to planned or unplanned outtages.
+
+To set up a load balancer with **active-active failover**:
+1. Create a load balancer with a single origin pool (`primary`) with multiple origins (`origin-1` and `origin-2`).
+1. For equal traffic, choose the same **Weight** for each origin. For guidance on other configurations, see [Weighted load balancers](../weighted-load-balancing).
+
+With this setup, your load balancer will direct all traffic to `primary`, which then directs traffic to `origin-1` and `origin-2` according to their respective weights. If enough origins become unhealthy that `primary` falls below its **Health Threshold**, traffic would then go to the **fallback pool**.
```
Comment (crwaters16) [inline products/load-balancing/src/content/understand-basics/load-balancers.md]: "become so unhealthy"? Feels like there's a missing word in the clause.

File: products/load-balancing/src/content/best-practices/index.md [hunk 2]
```diff
@@ -14,23 +14,21 @@ To avoid an inconsistent user experience when rolling out application changes, y
 
 To temporarily ignore an origin server:
 
-1. Deliberately configure a monitor so that your origin will fail health checks and Cloudflare will consider the origin server unhealthy and route traffic away from it. For example, set the **Response Code** to an HTTP status code you know your server does not return. Alternatively, you could set a value for **Response Body** that will not match. For example, you can configure the Response Body in a monitor to look for specific text.
-
-  ![](../static/images/best-practices-1.png)
+1. Update the associated monitor so that your origin will fail health checks — which will route traffic away from that origin. This might include setting:
```
Comment (crwaters16) [inline products/load-balancing/src/content/best-practices/index.md]: Consider replacing dash with comma (dash affects the pacing slightly)

**Review by crwaters16 — APPROVED** [review]
Non-critical comments in review, otherwise looks great!

**Pull request merged**